crates/eigen-cli/src/bls.rs

```diff
@@ -0,0 +1,60 @@
+use crate::args::BlsKeystoreType;
+use crate::EigenBlsKeyStoreError;
+use rust_bls_bn254::keystores::{pbkdf2_keystore::Pbkdf2Keystore, scrypt_keystore::ScryptKeystore};
+
+/// BlsKeystore types  
+pub enum BlsKeystore {
```

Comment:
this seems redundant with `BlsKeystoreType`

Reply:
my bad, the doc should have been just blskeystore, instead of types.